README.md

```diff
@@ -1,7 +1,7 @@
 # Yoti Go SDK
 
+[![Build Status](https://github.com/getyoti/yoti-go-sdk/workflows/Unit%20Test/badge.svg?branch=master)](https://github.com/getyoti/yoti-go-sdk/actions)
```

Comment:
The badge needs updating still:
```suggestion
[![Build Status](https://github.com/getyoti/yoti-go-sdk/workflows/Unit%20Tests/badge.svg?branch=master)](https://github.com/getyoti/yoti-go-sdk/actions)
```

Reply:
I thought I committed this suggestion already?